decidim-core/app/cells/decidim/redesigned_profile_cell.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Decidim
+  class RedesignedProfileCell < Decidim::ViewModel
+    include Decidim::ApplicationHelper
+    include Decidim::Core::Engine.routes.url_helpers
+    include Decidim::UserProfileHelper
+    include Decidim::AriaSelectedLinkToHelper
+    include Decidim::LayoutHelper
+    include ActiveLinkTo
+
+    delegate :current_organization, :current_user, :user_groups_enabled?, to: :controller
+    delegate :avatar_url, :nickname, :personal_url, :followers_count, :public_users_followings, :following_count, to: :presented_profile
+
+    TABS_ITEMS = {
+      activity: { icon: "bubble-chart-line", path: :profile_activity_path },
+      badges: { icon: "award-line", path: :profile_badges_path },
+      following: { icon: "user-shared-line", path: :profile_following_path },
```

Comment:
Now the different icons for following and followers is solved but I still find these confusing as they are so similar with each other. I'd expect the users to confuse these with each other easily. I would change these icons to clearly separate them from each other.

My suggestions would be:
- **Followers** - "eye" icon for "follows" since you are "watching" those items
- **Following** - "user-voice" icon indicating that you are signaling those people every time you do something publicly

As a side note, if it was up to me, I'd also list all the items that you are following here, not only users but I understand that it's not in the scope for the redesign and it also would require programming changes, not only design changes.

Reply:
Updated design with new icons:

- Following: `eye-2-line`

- Followers: `group-line`

![PX-icons_following_follower](https://user-images.githubusercontent.com/94045565/183634655-b5e8e3e0-2b95-4fd4-b785-16b5bea17943.png)

>  ... I'd also list all the items that you are following here, not only users ...

This is something we discussed with Product and it was decided to leave the list of users that the user is following and remove all other content.

Reply:
Great, I think these icons work really well. I was a bit afraid the followers would look too similar to groups but I think your icon selections work well and now they are clearly distinguishable from each other, at least to me.

> This is something we discussed with Product and it was decided to leave the list of users that the user is following and remove all other content.

Yeah, I kind of guessed it was something that was decided at product but I still disagree with them. There is currently really no place in the Decidim UI that would show all the things you are following, so it can become really hard to manage those for the users.

But as I said, this can be a discussion point for the future. I understand perfectly well that this is not something that is in the scope of the redesign process. I don't think it would not be too hard to implement that later on, if we can slowly bring a bit of user-centered thinking into the product team.